### PR TITLE
OVA1.0.4 Fixed Issues

### PR DESCRIPTION
**Pull request by TechWriter-Mayur**
Added the updates for OVA 1.0.4 and R2.
Signed-off-by: TechWriter-Mayur <mayur.gupta@seagate.com>

### CODE REVIEW

File: doc/CORTX_on_Open_Virtual_Appliance.rst [hunk 1]
```diff
@@ -54,18 +53,13 @@ The procedure to install CORTX on OVA is mentioned below.
 
 #. **For Oracle VM VirtualBox Users ONLY**:
```
Comment (mukul-seagate11) [inline doc/CORTX_on_Open_Virtual_Appliance.rst:54]: you can add drop down option for this section

Comment (TechWriter-Mayur) [reply doc/CORTX_on_Open_Virtual_Appliance.rst:54]: Added the drop-down option for Oracle VB.

File: doc/CORTX_on_Open_Virtual_Appliance.rst [hunk 2]
```diff
@@ -170,30 +208,30 @@ The procedure to install CORTX on OVA is mentioned below.
...
-      * put object
-      * delete all the above in reverse order
+         * create account
+         * create user
+         * create bucket
+         * put object
+         * delete all the above in reverse order
       
    
 #. Using the management IP from the **ip a l** command,  refer to these instructions to `configure the CORTX GUI <Preboarding_and_Onboarding.rst>`_.
```
Comment (mukul-seagate11) [inline doc/CORTX_on_Open_Virtual_Appliance.rst:237]: Instead of management IP, it should be public data IP

**Comment by mukul-seagate11** [comment]
Remove below lines as that is not required,

`If the SSPL service is inactive, run these command.

/opt/seagate/cortx/sspl/bin/sspl_setup post_install -p SINGLE

/opt/seagate/cortx/sspl/bin/sspl_setup config -f

systemctl start sspl-ll`

**Comment by mukul-seagate11** [comment]
Provide some better title to https://github.com/Seagate/cortx/blob/803f523c329983ff39f4fab54b2f93df7a4314f1/doc/testing_io.rst

**Comment by TechWriter-Mayur** [comment]
> Provide some better title to https://github.com/Seagate/cortx/blob/803f523c329983ff39f4fab54b2f93df7a4314f1/doc/testing_io.rst

Will the **Performing IO Operations Using S3 Client** title work?

**Comment by mukul-seagate11** [comment]
> > Provide some better title to https://github.com/Seagate/cortx/blob/803f523c329983ff39f4fab54b2f93df7a4314f1/doc/testing_io.rst
> 
> Will the **Performing IO Operations Using S3 Client** title work?

yes looks good

**Review by mukul-seagate11 — APPROVED** [review]
looks good & will merge the PR once QA is done with final validation

**Comment by mukul-seagate11** [comment]
@johnbent @gregnsk @ujjwalpl, please review & approve as OVA R1 & R2 Wiki has been updated & restructured accordingly

File: doc/OVA/1.0.4/CHANGELOG-1.0.4.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+-	Updated the IEM alert message.
+-	Optimized the object listing performance.
+-	Updated the SSPL support bundle temporary directory path.
+-	Fixed the issue of URL decode 'continuation-token' during List Object V2 request validation.
+-	Fixed the OVA for the host file and set the salt-master default configuration in case of a single node.
+-	Fixed the issue where the unboxing script is failing when the static IP network configuration option is selected.
+-	Fixed the issue where the S3 CLI requests fail with "No route to host" error intermittently.
+-	Updated the SSL certificates.
+-	Fixed the issue where SFTP session returns with non-zero exit code.
+-	Updated the glusterfs service relation with the salt-master.
```
Comment (ujjwalpl) [inline doc/OVA/1.0.4/CHANGELOG-1.0.4.md:149]: Updated the glusterfs service relation with the salt-master.

Such things can be filtered out as users wont be able to understand or make sense out of this.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

**Review by johnbent — APPROVED** [review]

**Review by johnbent — APPROVED** [review]

**Comment by AshwiniS-Borse** [comment]
Approved

**Review by AshwiniS-Borse — APPROVED** [review]

**Pull request merged**